manifests/l/LightburnSoftware/Lightburn/1.2.00/LightburnSoftware.Lightburn.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LightburnSoftware.Lightburn
+PackageVersion: 1.2.00
+PackageLocale: en-US
+Publisher: Lightburn Software
+PublisherUrl: https://Lightburnsoftware.com/
+PublisherSupportUrl: https://forum.Lightburnsoftware.com/
+# PrivacyUrl: 
+# Author: 
+PackageName: Lightburn
```

```suggestion
PackageName: LightBurn
```
This must match Control Panel for consistency.